orchard/Driver.py

```diff
@@ -0,0 +1,86 @@
+import sys
+import os.path
+from yaml import load
+import yaml
+try:
+    from yaml import CLoader as Loader
```

Comment:
This is only needed [for libyaml](http://pyyaml.org/wiki/LibYAML), so can drop to just a single import line of `from yaml import load, Loader, SafeDumper` or since there are so many variants happening just `import yaml` and do `yaml.load`, `yaml.Loader`, `yaml.SafeDumper` throughout the code.